public/embed-chart.html

```diff
@@ -199,6 +233,27 @@ <h3 class="subtitle">The partner application embeds selected Visier visuals into
             embeddingApp.on("error", function(msg) {
                 // Refer to the following list for common error cases.
                 switch (msg?.code) {
+                    /**
+                     * When this is a second page that was opened that embeds content for a different user or tenant,
+                     * the SDK attempts to clean up any prior sessions. This error is emitted in the case of unexpected
+                     * errors performing that clean up on the other page.
+                     *
+                     * The partner application can clean up the other session and attempt embedding once again.
+                     */
+                    case "FAILED_TO_EJECT_PREVIOUS_SESSION":
+                        showErrorModal("The user is logged into another tenant on another tab or window.");
+                        hideCharts();
+                        break;
+                    /**
+                     * This timeout error is emitted when there are unexpected issues with the authentication set up or if
```

Comment:
This timeout error emits if there are unexpected issues with the authentication set up or if the set up took longer than expected, such as due to network latency. Possible causes include: